proto/lbm/foundation/v1/foundation.proto

```diff
@@ -34,17 +34,33 @@ message Member {
   // address is the member's account address.
   string address = 1;
 
-  // participating is the flag which allows one to remove the member by setting the flag to false.
-  bool participating = 2;
-
   // metadata is any arbitrary metadata to attached to the member.
-  string metadata = 3;
+  string metadata = 2;
 
   // added_at is a timestamp specifying when a member was added.
   google.protobuf.Timestamp added_at = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
 }
 
-// ThresholdDecisionPolicy implements the DecisionPolicy interface
+// MemberRequest represents a foundation member to be used in Msg server requests.
+// Contrary to `Member`, it doesn't have any `added_at` field
+// since this field cannot be set as part of requests.
+message MemberRequest {
```

Comment:
This name make confuse for me, there is no query prefix but it looks like a query request. So how about changing `MemberUpdate` or others?

Reply:
As always, the message is taken from the upstream. Unless there is a critical issue, I think it would be better not to make a change.

Reply:
This is not a critical issue, but as I mentioned, it make me to confuse. 
BTW, could you share the link of similar upstream message of proto.

Reply:
> BTW, could you share the link of similar upstream message of proto.

https://github.com/cosmos/cosmos-sdk/blob/5f01f6f5e51156f8828e0b25e898d801442cd211/proto/cosmos/group/v1/tx.proto#L100
and
https://github.com/cosmos/cosmos-sdk/blob/5f01f6f5e51156f8828e0b25e898d801442cd211/proto/cosmos/group/v1/types.proto#L31-L44